<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteDebugDialog</class>
 <widget class="QDialog" name="RemoteDebugDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Remote debugging configuration</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="debugText">
      <property name="text">
       <string>Debugger:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="debuggerCombo">
      <item>
       <property name="text">
        <string>GDB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WinDbg - Pipe</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ipText">
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ipEdit">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="portText">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="portEdit">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="pathText">
      <property name="text">
       <string>Path:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="pathEdit">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>301</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RemoteDebugDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RemoteDebugDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
